src/storage/storage_structure/lists/lists_update_store.cpp

```diff
@@ -88,6 +102,46 @@ void ListsUpdateStore::insertRelIfNecessary(shared_ptr<ValueVector>& srcNodeIDVe
     }
 }
 
+void ListsUpdateStore::deleteRelIfNecessary(const shared_ptr<ValueVector>& srcNodeIDVector,
+    const shared_ptr<ValueVector>& dstNodeIDVector, const shared_ptr<ValueVector>& relIDVector) {
+    auto srcNodeID =
+        srcNodeIDVector->getValue<nodeID_t>(srcNodeIDVector->state->getPositionOfCurrIdx());
+    auto dstNodeID =
+        dstNodeIDVector->getValue<nodeID_t>(dstNodeIDVector->state->getPositionOfCurrIdx());
+    auto relID = relIDVector->getValue<int64_t>(relIDVector->state->getPositionOfCurrIdx());
+    auto tupleIdx = isNewlyInsertedRel(relID);
```

Comment:
Functions that start with "is" by convention return a bool. Maybe rename to getTupleIdxIfInsertedRel()?

Reply:
Renamed to `getTupleIdxIfInsertedRel`